### ISSUE TITLE
[Request] Add timestamp to !submissionInfo.txt

### ISSUE THREAD

**Issue by Intothisworld** [SabreTools/MPF#533]
**Is your feature request related to a problem? Please describe.**
When adding discs to DAT-o-matic for non-game, I have to check the modified date of the zipped log files to determine when the disc was dumped (or if MPF wasn't used, or the user has unpacked the logs and recompressed them, the date remains indeterminate. It would help expedite the process to have a timestamp for the dump available right in the generated submission info file. This would also help with general record-keeping for my own disc-dumping.

**Describe the solution you'd like**
A timestamp added to the submissionInfo.txt file. With Redumper, the timestamp can simply be pulled from the log. With DIC, it could simply be noted by MPF when the dump begins or ends or something.

**Describe alternatives you've considered**
Just accepting that some dumps will have indeterminate dates, and for the rest just being less lazy and checking the log file modified dates (and hoping that the date is reliable; it might not be evident sometimes that users have re-packed their log files. And sometimes users will use command line DIC/Redumper, make their own submissionInfo.txt, and then zip up the files in a way that makes it appear that MPF was the one to finalize everything.)

**Additional context**
None. Thanks, darksabre :)

**Comment by mnadareski** [comment]
I'm not 100% sure what this would accomplish, to be honest. I can include either a Unix timestamp or a `yyyy-MM-dd_hhmmss` sort of timestamp, but again, I don't know the "why" behind it. If it's to catch "fake" submissions, there's very little that can be done to fully prevent that.

**Comment by Intothisworld** [comment]
No, I didn't mean for fake submissions... I don't think this would do much to combat that. It would just overall be useful to know when exactly a disc was dumped. Redumper notes it in its logs, but DIC doesn't note it, which can be annoying. It would just be handy to have that info readily available (or available at all in DIC's case). My current method (checking the modified date of the zipped log files) is just a bit unreliable is all I was saying.

**Comment by mnadareski** [comment]
Discussed on Discord and it turns out the date is a field in No-Intro (used by Non-Game).
